.github/workflows/CI.yml

```diff
@@ -44,19 +44,39 @@ env:
   WIN_SIGNED_PKG_FOLDER: OTelCollectorAuthenticode/AuthenticodeSigner-SHA256-RSA
   SSM_PACKAGE_NAME: "testAWSDistroOTel-Collector"
   US_EAST_2_AMP_ENDPOINT: "https://aps-workspaces.us-east-2.amazonaws.com/workspaces/ws-1de68e95-0680-42bb-8e55-67e7fd5d0861"
+  TEST_FRAMEWORK_REPO: 'aws-observability/aws-otel-test-framework'
```

Comment:
can we use double quotes here and have
env variable : `TEST_FRAMEWORK_REPO` consistent across the workflows.

Reply:
I actually think the quotes aren't even necessary according to the [docs](https://docs.github.com/en/actions/learn-github-actions/expressions#example). I will probably end up removing them. I do think using one name is a good idea so I will update that.